<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>320</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" colspan="2">
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="page1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>验证：QQ邮箱是否合法</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="qqmail_input"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>结果</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="qqmail_output"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page2">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>从身份证获取出生年日和星座</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="id_input"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>结果</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="id_output"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page3">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>去重文字</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="run">
         <property name="text">
          <string>运行</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="text_result"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="spinBox"/>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>24</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>557</x>
     <y>8</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
